src/scss/components/pulumi-tertiary-nav.scss

```diff
@@ -0,0 +1,43 @@
+pulumi-tertiary-nav {
+    @apply bg-white text-base text-black;
+
+    ul {
+        @apply flex justify-evenly items-center flex-wrap text-base px-4;
+        list-style: none !important;
```

Comment:
Is this overriding something bone-headed I did elsewhere? (I tend to do this sort of thing and then regret it later...)